test/pylib/cpp/boost/prepare_instance.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+from __future__ import annotations
+
+import asyncio
+import logging
+import os
+import time
+from pathlib import Path
+
+from test.pylib.host_registry import HostRegistry
+from test.pylib.minio_server import MinioServer
+from test.pylib.s3_server_mock import MockS3Server
+from test.pylib.util import LogPrefixAdapter
+
+
+class PrepareWorker:
```

Comment:
Please add comments for the class itself and for its method. I think the name could be less generic.

Reply:
done